packages/doenetml-worker-javascript/src/utils/externalPathResolution.ts

```diff
@@ -0,0 +1,278 @@
+import {
+    FlatPathPart,
+    FlatIndex,
+    RefResolution,
+    Resolver,
+} from "lib-doenetml-worker";
+import { PublicDoenetMLCore } from "../CoreWorker";
+
+/**
+ * Attempts to resolve the component name `name` to a componentIdx,
+ * starting the search algorithm at node `origin`.
+ *
+ * Throws an error if the name is not resolved.
+ */
+export async function resolvePathImmediatelyToNodeIdx(
+    name: string,
+    coreWorker: PublicDoenetMLCore,
+    origin = 0,
+) {
+    console.log("resolve path immediately", { name, origin });
```

Comment:
This is here because it is a debug-only function? Still, if it is used in the tests it will print a lot.

Reply:
This was left here inadvertently.